app/controllers/users_controller.rb

```diff
@@ -1,7 +1,40 @@
 class UsersController < ApplicationController
 
   def new
+    @user = User.new
+    @categories = Category.all
+  end
+
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      redirect_to user_path(@user)
+    else
+      @categories = Category.all
+      render :new
+    end
```

Comment:
I think we should also remember to add some flash notices here so that the user gets a reason for seeing the new form again @bfpepper